pkg/cvo/status.go

```diff
@@ -367,6 +368,8 @@ func convertErrorToProgressing(history []configv1.UpdateHistory, now time.Time,
 	switch uErr.UpdateEffect {
 	case payload.UpdateEffectNone:
 		return uErr.Reason, fmt.Sprintf("waiting on %s", uErr.Name), true
+	case payload.UpdateEffectReport:
+		return uErr.Reason, fmt.Sprintf("waiting on %s", uErr.Name), false
```

Comment:
We're not actually waiting on these, right?  If our most serious error was a report-level issue, we completed the sync loop and should go `Progressing=False`.